queries.sql

```diff
@@ -31,4 +31,52 @@ WHERE NAME <> 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;  
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;  
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals
+
+UPDATE animals
+SET species = 'pokemon';
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] This query should not set the species for all animals to `pokemon`. As per the requirements, you should do the following:

   - UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.

   - UPDATE the animals table by setting the species column to pokemon for all animals that don't have species already set.
  - Commit the transaction.`

Reply:
resolved 👍